platform-cloud/docs/monitoring/run-details.md

```diff
@@ -0,0 +1,262 @@
+- 
+- [Run information](#run-information) with real-time Nextflow execution details
+- [General summary](#general-summary) and [task status](#task-status)
+- List of pipeline 
+- [Aggregated stats](#aggregate-stats), [load](#load), and [utilization](#utilization)
+
+<Tabs>
+<TabItem value="Tasks" label="Tasks" default>
+
+![Task status overview](./_images/task-status-tiles.png)
+
+The cards at the top of the **Tasks** tab provide a real-time status of all tasks in the pipeline run:
+
+- **Pending**: The task has been created, but not yet submitted to an executor.
+- **Submitted**: The task has been submitted to an executor, but is not yet running.
+- **Running**: The task has been launched by an executor (the precise definition of "running" may vary for each executor).
+- **Cached**: A previous (and valid) execution of the task was found and used instead of executing the task again. See [Cache and resume](../launch/cache-resume.mdx).
+- **Succeeded**: The task completed successfully.
+- **Failed**: The task failed.
+- **Aborted**: The task was submitted, but the run was cancelled or failed before the task could begin.
+
+### Processes 
+
+The **Processes** panel displays the status of each process in a pipeline run. In Nextflow, a process is an individual step in a pipeline, while a task is a particular invocation of a process for given input data. In the panel, each process is shown with a progress bar indicating how many tasks have been completed for that process.
+
+The progress bar is color-coded based on task status (**created**, **submitted**, **completed**, **failed**).
+
+Select a process to navigate to the [Tasks](#tasks) panel and filter the table contents by the selected process.
+
+### Tasks 
+
+The **Tasks** panel shows all the tasks that were executed in the run.
+
+Use the search bar to filter tasks with substrings in the table columns such as **process**, **tag**, **hash**, and **status**. For example, if you enter `succeeded` in the **Search task** field, the table displays only tasks that succeeded.
+
+#### Task details
+
+![Task details](./_images/task-details.png)
+
+Select a task in the task table to open the **Task details** dialog. The dialog has three tabs:
+
+- **About**
+  - **Name**: Process name and tag
+  - **Command**: Task script, defined in the pipeline process
+  - **Status**: Exit code, task status, attempts
+  - **Work directory**: Directory where the task was executed
+  - **Environment**: Environment variables supplied to the task
+  - **Execution time**: Metrics for task submission, start, and completion time
+  - **Resources requested**: Metrics for the resources requested by the task
+  - **Resources used**: Metrics for the resources used by the task
+
+- **Execution log**
+
+  The **Execution log** tab provides a real-time log of the selected task's execution. Task execution and other logs (such as `stdout` and `stderr`) are available for download from here, if still available in your compute environment.
```

Comment:
```suggestion
  The **Execution log** tab provides a real-time log of the selected task's execution. Task execution and other logs (such as `stdout` and `stderr`) are available for download if they are still available in your compute environment.
```
Not a blocker, but can "still available in your compute environment" be clarified? Should when they are not available be described?